docs/sources/helm-charts/mimir-distributed/run-production-environment-with-helm/monitor-system-health.md

```diff
@@ -16,132 +16,11 @@ refs:
 
 # Monitor the health of your system
 
-You can monitor Grafana Mimir or Grafana Enterprise Metrics itself, by collecting metrics and logs from Mimir or GEM that is running on a Kubernetes cluster. This is called _metamonitoring_.
+You can monitor Grafana Mimir or Grafana Enterprise Metrics by collecting metrics and logs from a Mimir or GEM instance that's running on a Kubernetes cluster. This process is called _metamonitoring_.
 
-> **Note:** In Grafana, you can create dashboards and receive alerts about those metrics and logs. To set up dashboards and alerts,
-> see [Installing Grafana Mimir dashboards and alerts] or [Grafana Cloud: Self-hosted Grafana Mimir integration](/docs/grafana-cloud/monitor-infrastructure/integrations/integration-reference/integration-mimir/).
+As part of *metamonitoring*, You can create dashboards and receive alerts about the metrics and logs collected from Mimir. To set up these dashboards and alerts,
+refer to [Installing Grafana Mimir dashboards and alerts](https://grafana.com/docs/mimir/<MIMIR_VERSION>/manage/monitor-grafana-mimir/installing-dashboards-and-alerts/).
 
-Alternatively, to monitor the health of your system without using the Helm chart, see [Collect metrics and logs without the Helm chart].
+You can also use the self-hosted Grafana Cloud integration to monitor your Mimir system. Refer to [Grafana Cloud: Self-hosted Grafana Mimir integration](/docs/grafana-cloud/monitor-infrastructure/integrations/integration-reference/integration-mimir/) for more information.
 
-## Configure the Grafana Agent operator via the Helm chart
```

Comment:
@eamonryan could you please confirm that we want to remove all of this? This leaves this topic pretty sparse, but this might be necessary until we determine the direction of the Mimir Helm chart. I'm definitely open to ideas for making this content more robust.

Reply:
Since it looks like Helm chart is being updated, I'll leave this content in with a deprecation notice for now. This will make it easier to update later.